docker/cuda/Dockerfile-9.0

```diff
@@ -6,19 +6,18 @@ RUN apt-get update && apt-get install -y \
     cmake \
     build-essential \
     curl \
+    lcov \
     pkg-config \
     openmpi-bin libopenmpi-dev \
-    lcov \
     libfftw3-dev \
     libboost-dev libboost-serialization-dev libboost-mpi-dev libboost-filesystem-dev libboost-test-dev \
     libgsl-dev \
     cython3 python3 python3-numpy python3-h5py \
-    lcov \
-    curl \
```

Comment:
are you sure that we don't need the curl package?

Reply:
see line 8